cocos2d/CCScheduler.m

```diff
@@ -137,19 +300,20 @@ -(void)setPaused:(BOOL)paused
 @end
 
 
-@interface NSNull(CCSchedulerTarget)<CCSchedulerTarget>
+@interface NSNull(CCSchedulableTarget)<CCSchedulableTarget>
 @end
 
 
-@implementation NSNull(CCSchedulerTarget)
+@implementation NSNull(CCSchedulableTarget)
 -(NSInteger)priority {return NSIntegerMax;}
 @end
 
 
+//MARK: CCTimer
```

Comment:
In xcode, `#pragma mark` and `//MARK` do the same thing. Consistency would be good. There's 692 `#pragma mark`s and 120 `//MARK:`s. I'll change the comments to pragmas for consistency, and I'll skip the ones in chipmunk (which looks like most of them)